sdks/java/io/hadoop-common/build.gradle

```diff
@@ -17,7 +17,7 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature()
+applyJavaNature(automaticModuleName: 'beam.sdks.java.io.hadoop.common')
```

Comment:
This is a valid name, but I somehow expected it to coincide with the package definition in the code, currently `org.apache.beam.sdk.io.hadoop` I think is ok as it is, but maybe someone else can argue pro being consistent.

Reply:
The reason I named it "common" is because it holds some commons so I didn't want to block the hadoop name for the future.